src/cmd/sync.rs

```diff
@@ -86,8 +103,8 @@ pub fn run(args: &Args) -> Result<()> {
 
 /// Version string can represent either cros repo version or an arc version.
 /// This function detects which and extracts its appropriately from the args.
-fn extract_version(args: &Args) -> Result<String> {
-    let version = if !args.arc {
+fn extract_version(args: &Args, is_arc: &bool) -> Result<String> {
```

Comment:
This may be a little off topic but, since we have separated the options for specifying directories into cros and arc, it may be a good idea to separate these functions (`extract_version()`, `prepare_repo_paths()`) as well.

Reply:
sgtm! I think we should do that. Happy to review the pull requests for that change. Thank you!